src/components/dapp-staking/dapp/Dapp.vue

```diff
@@ -1,5 +1,6 @@
 <template>
   <div v-if="dapp">
+    <back-to-page :text="$t('dappStaking.dappPage.back')" :link="Path.DappStaking" />
```

Comment:
How about changing here to `Back to dApps list` ?

```suggestion
    <back-to-page :text="$t('dappStaking.stakePage.backToDappList')" :link="Path.DappStaking" />
```

![image](https://user-images.githubusercontent.com/92044428/200571259-ce344ad6-ae6c-4ce0-a079-8f1fe2412151.png)

Reply:
Sure thanks!

Reply:
Resolved all